CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- `heightImg` and `widthImg` to `SkuSelector`.
```

Comment:
refrain from abbreviating unless it'd be extremely long otherwise.

I suggest renaming these props to `imageWidth`/`imageHeight`, or even maybe `previewWidth` and `previewHeight`